code/game/atom/alternate_appearance.dm

```diff
@@ -212,3 +212,10 @@ GLOBAL_LIST_EMPTY(active_alternate_appearances)
 	return ..()
 
 /datum/atom_hud/alternate_appearance/basic/food_demands
+
+/datum/atom_hud/alternate_appearance/basic/heretic
```

Comment:
We changed tears in reality to be seen by ghosts. This should be no different, so check the code, because we may have alternate appearances to show to heretics and ghosts.

Reply:
Just checked, works as intended; only heretics are capable of seeing the mark.